docs/upgrade/2.0-3.0.md

```diff
@@ -0,0 +1,314 @@
+# Migration Guide, Version 2.0 to 3.0
+
+This document aims to serve as a guide for manually migrating your existing
+Madison site that is lower than version 3.0, up to the 3.0 version. There have
+been many changes, but below you can find a list of these changes as well as
+steps you can follow to upgrade to the current version.
+
+**If you wish to avoid the manual process, you can use the new Chef and Capistrano
+setup for provisioning a server and deploying to it.** To do this, make sure to
+back up your database for migration, then use [the deployment documentation
```

Comment:
Should they also grab a copy of their `.env` file? Will they need some settings from there?